site/content/en/docs/manual/advanced/projects.md

```diff
@@ -2,17 +2,117 @@
+
+   ![Create new project window](/images/create_new_project.png)
+
+2. Go to the **Constructor** tab.
+3. In the **Name** field enter the name of your project.
+4. Click **Add label** and fill in the following fields:
+
+  1. **Label name** Mandatory field. Enter the name of the annotation label.
+  2. **Change colour of the label** (![change colour of the label](/images/color-field.png)) Optional field. Click the icon and select the color from the color palette.
+  3. **Add attribute** Optional field. Opens a menu, where you can set the following attributes:
+     1. **Name** Mandatory field. Enter the name of the attribute.
+     2. From the drop-down list, select an HTML element, representing the attribute:
+        - **Select** is a placeholder and cannot be used as an attribute.
+        - **Radio** Use it to add a radio button, to select only one option from several.
+        - **Checkbox** Use it to add a checkbox for multiple options choice.
+        - **Text** Use it for text attributes.
+        - **Number** Use it for numeric attributes.
+
+   3. In the **Attribute value** field, specify the attribute value. This field depends on the type of the HTML element from the previous step:
+
+       - For **Radio** enter one or more text values. To separate values, press Enter after each value name. To remove value from the field, at the end of the value name, click `X`.
+       - For **Checkbox** from the drop-down list select one of the values **True** or **False**.
+       - For **Text** enter the default value in a text form. You can specify only one default value.
+       - For **Number** specify a range, separated by a semicolon, from the lowest number to the highest number, and step.  For example: `10;100;2`
+   4. If you want to change the attribute from frame to frame, select the **Mutable** checkbox.
+   5. If you want to remove the attribute, click ![Garbage bin](/images/garbage-bin.png)
+
+      > **Note** you can add more than one attribute, by clicking **Add attribute**  more than one time.
+
+      ![Create project-filled labels and attributes](/images/create_new_project_01.png)
+
+5. Click **Continue**. Now the project has a label. To add more labels repeat steps 1 to 5.
+
+   ![Create project filled labels and attributes](/images/create_new_project_02.png)
+
+6. Optional. Use **Advanced configuration** to add an issue tracker, source storage, and target storage to the project:
+
+   1. Click the **Advanced configuration** drop-down.
+   2. In the **Issue tracker** field, enter the URL of the issue tracker, that you use.
+   3. In the **Source storage** field enter the source storage from which you import resources.
+   4. In the **Target storage** field enter the target storage to export resources.
+7. Click:
+   - In case you want to open a new project **Submit & open**
+   - In case you want to create another project **Submit & continue**
+
+The project will appear on the project pane.
```

Comment:
Did you mean page?

Reply:
No, I did mean "pane" because I've marked what is "pane" at the beginning of the procedure description.